apps/ledger-live-desktop/tests/specs/support/unsupported-os.spec.ts

```diff
@@ -15,6 +15,10 @@ test("Unsupported OS", async ({ page }) => {
   await test.step("displays the error page", async () => {
     await layout.renderError.waitFor({ state: "visible" });
     await expect(layout.renderError).toBeVisible();
-    await expect(await layout.page.screenshot()).toMatchSnapshot("error-os-unsupported.png");
+    await expect(
+      await layout.page.screenshot({
+        mask: [page.locator("[data-test-id='stacktrace']")],
```

Comment:
Shouldn't the `page.locator` live in the `Layout` model ?

Reply:
No, it comes from `tests/fixtures/common.ts` and `@playwright/test` package.

Reply:
That doesn't answer my question, we're suppose to put all `page.locator` calls in the constructor of a model, I saw that everywhere, why not for this `page.locator` ?

Reply:
My bad, I didn’t understand that you were mentioning this line, of course locators should live in the page object, it’s a good practice.